packages/datadog-plugin-jest/src/util.js

```diff
@@ -47,10 +50,46 @@ function getJestTestName (test) {
   return titles.join(' ')
 }
 
+function isMarkedAsUnskippable (test) {
+  let docblocks
+
+  try {
+    const testSource = readFileSync(test.path, 'utf8')
+    docblocks = parse(extract(testSource))
+  } catch (e) {
+    // If we have issues parsing the file, we'll assume no unskippable was passed
+    return false
+  }
+
+  // docblocks were correctly parsed by it does not include a @datadog block
+  if (!docblocks?.datadog) {
+    return false
+  }
+
+  try {
+    return JSON.parse(docblocks.datadog).unskippable
+  } catch (e) {
+    // If the @datadog block comment is present but malformed, we'll run the suite
+    return true
```

Comment:
Does it make sense, perhaps, to log the error here? Otherwise the customer might not be aware that they have a malformed `docblocks.datadog` block.

Reply:
yeah that makes sense